src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -200,6 +201,14 @@ export const SendAssetsForm = ({ txConfig, onComplete, protectTxButton }: ISendF
   const [isResolvingName, setIsResolvingDomain] = useState(false); // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
   const [fetchedNonce, setFetchedNonce] = useState(0);
   const [isSendMax, toggleIsSendMax] = useState(false);
+  const [selectedNetwork, setSelectedNetwork] = useState<Network>(getNetworkById(DEFAULT_NETWORK));
+
+  const TO_FIELD_ERROR = useMemo(() => {
```

Comment:
I think we can add the network to `TO_FIELD_ERROR` as so: `Enter a valid $network address, ENS name, or blockchain domain`, to reduce the clutter here?